src/components/SuccessAlert.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const SuccessAlert = () => {
+  return (
+    <div
+      className="p-4 mb-4 text-sm text-green-800 rounded-lg bg-green-50 dark:bg-gray-800 dark:text-green-400"
+      role="alert"
+    >
+      <span className="font-medium">Awesome!</span>
+      You Successfully create an expense group
```

Comment:
same thing here, or you can just give some space between them

Reply:
@parykhan-jameel I changed it to one sentence.